### PR TITLE
use useScreenShare hook - Daily react hooks

### PR DESCRIPTION
**Pull request by harshithpabbati**
This PR is to make a screen share context and use the screenshare hook.

### CODE REVIEW

File: custom/shared/contexts/ScreenShareProvider.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+export const ScreenShareProvider = ({ children }) => {
+  const {
+    isSharingScreen,
+    screens,
+    startScreenShare,
+    stopScreenShare
+  } = useDailyScreenShare();
+
+  const isDisabled = useMemo(() => screens.length >= MAX_SCREEN_SHARES && !isSharingScreen,
```
Comment (nienkedekker) [inline custom/shared/contexts/ScreenShareProvider.js:17]: Isn't `screens` an [object](https://docs.daily.co/reference/daily-react-hooks/use-screen-share#screens-properties)?  Will `.length`? work?

Comment (jayne-mast) [reply custom/shared/contexts/ScreenShareProvider.js:17]: I think it's an array: https://github.com/daily-co/daily-react-hooks/blob/main/src/hooks/useScreenShare.ts#L59

Comment (harshithpabbati) [reply custom/shared/contexts/ScreenShareProvider.js:17]: yep, it's an array of screen objects.

Comment (nienkedekker) [reply custom/shared/contexts/ScreenShareProvider.js:17]: But then why does [this](https://docs.daily.co/reference/daily-react-hooks/use-screen-share#screens-properties) say object 🥲 is there a different `screens` I should be looking at? Or am I just misunderstanding completely lol

Comment (harshithpabbati) [reply custom/shared/contexts/ScreenShareProvider.js:17]: Yeah, I think it's a typo in the docs, we should we change that.

Comment (nienkedekker) [reply custom/shared/contexts/ScreenShareProvider.js:17]: thanks for clarifying!

**Review by nienkedekker — APPROVED** [review]

File: custom/shared/contexts/ScreenShareProvider.js (hunk 1 above)
Comment (jayne-mast) [reply custom/shared/contexts/ScreenShareProvider.js:17]: We should also update the JSON example then. I'll make a Linear ticket.

**Review by lizadaily — APPROVED** [review]
lgtm

**Pull request merged**